docs/virtual_machines/templates.md

```diff
@@ -6,15 +6,15 @@ KubeVirt on OpenShift).
 
 By deploying KubeVirt on top of OpenShift the user can benefit from the
 [OpenShift
-Template](https://docs.openshift.org/latest/dev_guide/templates.html)
+Template](https://docs.openshift.com/container-platform/4.6/virt/vm_templates/virt-creating-vm-template.html)
```

Comment:
The rest of the docs refer to OKD. Can we link OKD here too?

Reply:
Sure can